From: Qingqing <wangyiqing@kylinos.cn>
Date: Mon, 17 Aug 2026 02:51:42 +0000
Subject: Translated using Weblate (Russian)

Currently translated at 0.0% (0 of 2 strings)

Translation: openkylin-huanghe/peony-admin
Translate-URL: http://weblate.openkylin.top/projects/openkylin-huanghe/peony-admin/ru/
---
 peony-admin/translations/peony-admin_ja.ts         | 18 +++++++++++++
 peony-admin/translations/peony-admin_ru.ts         |  4 +--
 .../translations/peony-bluetooth-plugin_ja.ts      | 30 ++++++++++++++++++++++
 3 files changed, 50 insertions(+), 2 deletions(-)
 create mode 100644 peony-admin/translations/peony-admin_ja.ts
 create mode 100644 peony-bluetooth-plugin/translations/peony-bluetooth-plugin_ja.ts

diff --git a/peony-admin/translations/peony-admin_ja.ts b/peony-admin/translations/peony-admin_ja.ts
new file mode 100644
index 0000000..437cdc8
--- /dev/null
+++ b/peony-admin/translations/peony-admin_ja.ts
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="ja">
+<context>
+    <name>Peony::AdminMenuPlugin</name>
+    <message>
+        <location filename="../admin-menu-plugin.cpp" line="61"/>
+        <location filename="../admin-menu-plugin.cpp" line="87"/>
+        <source>Open Directory as Admin</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../admin-menu-plugin.cpp" line="106"/>
+        <source>Open Text as Admin</source>
+        <translation type="unfinished"></translation>
+    </message>
+</context>
+</TS>
diff --git a/peony-admin/translations/peony-admin_ru.ts b/peony-admin/translations/peony-admin_ru.ts
index 80fa70b..120bcf9 100644
--- a/peony-admin/translations/peony-admin_ru.ts
+++ b/peony-admin/translations/peony-admin_ru.ts
@@ -7,12 +7,12 @@
         <location filename="../admin-menu-plugin.cpp" line="61"/>
         <location filename="../admin-menu-plugin.cpp" line="87"/>
         <source>Open Directory as Admin</source>
-        <translation type="unfinished"></translation>
+        <translation>Открыть каталог от имени администратора</translation>
     </message>
     <message>
         <location filename="../admin-menu-plugin.cpp" line="106"/>
         <source>Open Text as Admin</source>
-        <translation type="unfinished"></translation>
+        <translation>Открыть файл от имени администратора</translation>
     </message>
 </context>
 </TS>
diff --git a/peony-bluetooth-plugin/translations/peony-bluetooth-plugin_ja.ts b/peony-bluetooth-plugin/translations/peony-bluetooth-plugin_ja.ts
new file mode 100644
index 0000000..6750e00
--- /dev/null
+++ b/peony-bluetooth-plugin/translations/peony-bluetooth-plugin_ja.ts
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="ja">
+<context>
+    <name>Peony::BluetoothPlugin</name>
+    <message>
+        <location filename="../bluetoothplugin.cpp" line="165"/>
+        <source>Send from bluetooth to...</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../bluetoothplugin.h" line="54"/>
+        <source>Peony-Qt bluetooth Extension</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../bluetoothplugin.h" line="55"/>
+        <source>Bluetooth Menu Extension</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <source>bluetooth Menu Extension</source>
+        <translation type="vanished">蓝牙菜单扩展</translation>
+    </message>
+    <message>
+        <source>bluetooth Menu Extension.</source>
+        <translation type="vanished">蓝牙菜单扩展.</translation>
+    </message>
+</context>
+</TS>
